client/client.go

```diff
@@ -12,83 +15,98 @@ import (
 	"github.com/sajari/docconv"
 )
 
-var (
-	defaultParam    = "input"
```

Comment:
Ah, I'm not looking in the right place then.  I thought this was it:

https://github.com/sajari/docconv/blob/master/docd/main.go#L112